addons/website_slides/controllers/main.py

```diff
@@ -489,6 +512,16 @@ def channel(self, channel, category=None, tag=None, page=1, slide_category=None,
         elif uncategorized:
             query_string = "?search_uncategorized=1"
 
+        errors = {'access_error': False}
+        if request.params.get('access_error') == 'course_content' and request.params.get('access_error_slide_id'):
+            # Access are re-verified to support use case where the user refresh the page after an update of their access
+            user_slide_authorization = self._get_user_slide_authorization(int(request.params.get('access_error_slide_id')))
+            if user_slide_authorization.get('status') == 'not_authorized':
```

Comment:
No need to use ``.get`` as you always have this key available in the returned dict. You can access directly the key. Get is used when you are unsure the key exists and you don't want to handle errors.

Reply:
Thanks, indeed it gives more hint about the code (= that the key is always present at that line)